src/ui/theme/shadows.ts

```diff
@@ -1,14 +1,20 @@
 export const SHADOWS = {
```

Comment:
Can we just add shadows in semantic tokens? I think it's the correct places to have the shadows and supports having light/dark mode variations.

Reply:
These are already in the semantic tokens.